spec/lib/price_history_spec.rb

```diff
@@ -23,20 +23,55 @@
 
     history = PriceHistory.call(package: basic, year: "2020")
     expect(history).to eq(
-      "Göteborg" => [100_00],
-      "Stockholm" => [30_00, 40_00],
-    )
+                         "Göteborg" => [100_00],
```

Comment:
Is this change an intentional formatting update?

Reply:
@ziadsawalha Yes, As for the new test cases, I've used my preset formatting so I've changed this one to match the file in the same format. Otherwise there is no need to do this.